### PR TITLE
Drupal 9 compatibility fixes

### PR DESCRIPTION
**Pull request by shashank-naik**
Hi @malaynayak 

Thank you for working on this module. I tested this out on one of my projects running Drupal 9.4 and confirmed that it worked after a couple of minor tweaks. This PR captures all my changes.

Thanks!

### CODE REVIEW

File: src/Form/EntityDeleteLogSeetingsForm.php [hunk 1]
```diff
@@ -76,7 +76,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     // Add some helpful links.
     $help_links = [
       Link::fromTextAndUrl('View the Entity Delete Log README',
-          Url::fromUserInput('/' . drupal_get_path('module', 'entity_delete_log') . '/README.txt')),
+          Url::fromUserInput('/' . \Drupal::service('extension.path.resolver')->getPath('module', 'entity_delete_log') . '/README.txt')),
```
Comment (malaynayak) [inline src/Form/EntityDeleteLogSeetingsForm.php]: @shashank-naik Thanks for working on this. Can we use DI here instead of static call.

**Review by malaynayak — CHANGES_REQUESTED** [review]
@shashank-naik Thanks for working on this. Can we use DI here instead of static call.

File: src/Form/EntityDeleteLogSeetingsForm.php (hunk 1 above)
Comment (shashank-naik) [reply src/Form/EntityDeleteLogSeetingsForm.php]: Yeah, it should be possible. I'll look into it.

Comment (shashank-naik) [reply src/Form/EntityDeleteLogSeetingsForm.php]: I've made this change. Please review when you can. I've also corrected a minor typo in the classname. Thanks!

**Comment by malaynayak** [comment]
@shashank-naik a new release has been published for https://www.drupal.org/project/entity_delete_log. U can use that instead of this repo.

**Pull request merged**